test/libsolidity/semanticTests/variables/transient_state_variable.sol

```diff
@@ -0,0 +1,22 @@
+contract C {
+    uint transient public x;
+
+    function f() public {
+        x = 8;
+    }
+    function g() public returns (uint) {
+        x = 0;
+        f();
```

Comment:
```suggestion
        this.f();
```

Reply:
By the way @ekpyron, is `this` implicit here (i.e. is it even necessary), and if no - what is the difference between `f()` and `this.f()`?

Reply:
`f()` is an internal call and `this.f()` is an external one. `f()` is a public function so both are possible and valid, they'll just do different things.

Though the way `f()` is implemented now (just modifying a transient variable) there would be no difference in test output.

Reply:
Thanks - Daniel already answered this in one of our Mon/Wed calls :)